core/controlplane/config/templates/cloud-config-worker

```diff
@@ -642,6 +664,9 @@ write_files:
       docker run --rm --net=host \
         -v /etc/kubernetes:/etc/kubernetes \
         -v /etc/resolv.conf:/etc/resolv.conf \
+        {{- if .Experimental.ClusterTLSBootstrap.Enabled }}
+        -v /var/run/kubernetes:/var/run/kubernetes \
+        {{- end }}
```

Comment:
If we go [that](https://github.com/kubernetes-incubator/kube-aws/pull/449/files#r108824744) way, this block can be just:

```
-v /var/run/kubernetes:/var/run/kubernetes
```

without a `if`.